src/DiContainer/Attributes/Service.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Kaspi\DiContainer\Attributes;
+
+#[\Attribute(\Attribute::TARGET_CLASS)]
+final class Service
+{
+    public function __construct(public string $id) {}
```

Comment:
Add throw if class not exist.
throw new AutowiredException("Class [$this->id] not exist");